tutorials/darcy_thermo_mech/step03_darcy_material/include/kernels/DarcyPressure.h

```diff
@@ -31,7 +31,7 @@ class DarcyPressure : public ADKernel
   /// The permeability. Note that this is declared as a \p MaterialProperty. This means that if
   /// calculation of this property in the producing \p Material depends on non-linear variables, the
   /// derivative information will be lost here in the consumer and the non-linear solve will suffer
-  const ADMaterialProperty<Real> & _permeability;
+  const MaterialProperty<Real> & _permeability;
```

Comment:
Changing this to `MaterialProperty` to fit the above descriptive comment. If this change is undesirable, the comment needs to be updated.